portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,28 +45,27 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
    PreparedQuery results = datastore.prepare(query);
    
-   List<Task> task = new ArrayList<Task>();
+   List<Task> taskList = new ArrayList<Task>();
    for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
+      String email = (String) entity.getProperty("email");
```

Comment:
Seems like this isn't really a "task" anymore, more like a comment maybe? could rename the class